src/index.html

```diff
@@ -2,9 +2,8 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <title>Paratii</title>
+    {{meta}}
```

Comment:
If we are calling this `meta` then should every `<meta>` tag be moved into a variable? We still have `<meta name="viewport">` below

Reply:
ok